qa/tasks/cephfs/caps_helper.py

```diff
@@ -66,14 +66,16 @@ def gen_mds_cap_str(caps):
 
     caps = ((perm1, fsname1, cephfs_mntpt1), (perm2, fsname2, cephfs_mntpt2))
     """
-    def _gen_mds_cap_str(perm, fsname=None, cephfs_mntpt='/'):
+    def _gen_mds_cap_str(perm, fsname=None, cephfs_mntpt='/', root_squash=''):
         mds_cap = f'allow {perm}'
         if fsname:
             mds_cap += f' fsname={fsname}'
         if cephfs_mntpt != '/':
             if cephfs_mntpt[0] == '/':
                 cephfs_mntpt = cephfs_mntpt[1:]
             mds_cap += f' path={cephfs_mntpt}'
+        if root_squash != '':
+            mds_cap += f' root_squash'
```

Comment:
since the string here is definite, the `f` can be removed from here

Reply:
That's failing the test: ./tasks/cephfs/caps_helper.py:78:24: F541 f-string is missing placeholders

Reply:
@lxbsz ping?

Reply:
@dparmar18 @joscollin @vshankar Sorry, I just missed these comments for some reasons.

I will update it. Thanks!